tests/components/stookwijzer/test_config_flow.py

```diff
@@ -9,35 +12,62 @@
 from homeassistant.data_entry_flow import FlowResultType
 
 
-async def test_full_user_flow(hass: HomeAssistant) -> None:
+async def test_full_user_flow(
+    hass: HomeAssistant,
+    mock_stookwijzer: MagicMock,
+    mock_setup_entry: AsyncMock,
+    snapshot: SnapshotAssertion,
+) -> None:
     """Test the full user configuration flow."""
     result = await hass.config_entries.flow.async_init(
         DOMAIN, context={"source": SOURCE_USER}
     )
 
-    assert result.get("type") is FlowResultType.FORM
-    assert result.get("step_id") == "user"
-    assert "flow_id" in result
-
-    with patch(
-        "homeassistant.components.stookwijzer.async_setup_entry", return_value=True
-    ) as mock_setup_entry:
-        result2 = await hass.config_entries.flow.async_configure(
-            result["flow_id"],
-            user_input={
-                CONF_LOCATION: {
-                    CONF_LATITUDE: 1.0,
-                    CONF_LONGITUDE: 1.1,
-                }
-            },
-        )
-
-    assert result2.get("type") is FlowResultType.CREATE_ENTRY
-    assert result2.get("data") == {
-        "location": {
-            "latitude": 1.0,
-            "longitude": 1.1,
-        },
-    }
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "user"
+
+    result2 = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        user_input={CONF_LOCATION: {CONF_LATITUDE: 1.0, CONF_LONGITUDE: 1.1}},
+    )
+
+    assert result2["type"] == FlowResultType.CREATE_ENTRY
+    assert result2 == snapshot
```

Comment:
I personally don't like snapshot tests in the config flow as it hides away what you're actually testing. Not a blocker but I do want to mention it

Reply:
Yeah maybe... Let's adjust.